src/Microsoft.Health.Dicom.SqlServer/Features/ExtendedQueryTag/ExtendedQueryTagDataRowsBuilder.cs

```diff
@@ -91,6 +105,18 @@ private static void AddDateTimeRow(DicomDataset instance, List<InsertDateTimeExt
             }
         }
 
+        private static void AddDateTimeRow(DicomDataset instance, List<InsertDateTimeExtendedQueryTagTableTypeV2Row> dateTimeRows, QueryTag queryTag)
```

Comment:
were you going to add a qualifier here, like `...WithUtc`?

Reply:
I was debating, but left it as an overload of AddDateTimeRow. If there is a preference, I don't mind making the change.

Reply:
Updated in the latest iteration.